content/style_html.md

```diff
@@ -0,0 +1,260 @@
++++
+title = "HTML and CSS Guide"
```

Comment:
We should probably change this to Shortcodes since we're not documenting HTML or CSS.
```suggestion
title = "Shortcodes"
```